flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontendOptions.java

```diff
@@ -94,6 +94,15 @@ public class CliFrontendOptions {
                                     + "program that was part of the program when the savepoint was triggered.")
                     .build();
 
+    public static final Option FLINK_CONFIG =
+            Option.builder("fc")
```

Comment:
[SQL Client](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sqlclient/#sql-client-startup-options) provides `-D` to pass extra Flink options dynamically. Maybe we can follow the same naming here?